src/lib/core/components/visualizations/implementations/MapVisualization.tsx

```diff
@@ -531,14 +601,6 @@ function MapViz(props: VisualizationProps) {
     [updateVizConfig]
   );
 
-  const handleOutputEntityChange = useCallback(
```

Comment:
Here (and below) I bit the bullet and removed some disabled functionality that @chowington took care of.